examples/stringification/main.cpp

```diff
@@ -0,0 +1,432 @@
+#define DOCTEST_CONFIG_IMPLEMENT_WITH_MAIN
+#include <doctest/doctest.h>
+
+DOCTEST_MAKE_STD_HEADERS_CLEAN_FROM_WARNINGS_ON_WALL_BEGIN
+// Pass
+#if __cplusplus >= 202002L
```

Comment:
This is broken on MSVC, it doesn't define `__cplusplus` properly.

Reply:
See my stringification branch for fix

Reply:
See CMakeLists.txt. It is already handled
```cmake
if(MSVC)
    # See https://devblogs.microsoft.com/cppblog/msvc-now-correctly-reports-__cplusplus/
    target_compile_options(stringification PUBLIC "/Zc:__cplusplus")
endif()
```

Reply:
https://github.com/Saalvage/doctest/blob/a9a9c23083111dac60afd7810f18ef9a762f2b36/doctest/doctest.h#L91

Reply:
It might be not ideal solution for a header-only library, but for test executable it should work just fine